src/main/java/uk/gov/hmcts/sptribs/ciccase/tab/CaseTypeTab.java

```diff
@@ -103,6 +121,13 @@ private void buildNotesTab(ConfigBuilder<CaseData, State, UserRole> configBuilde
             .field(CaseData::getNotes);
     }
 
+    private void buildBundlesTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        configBuilder.tab("bundles", "Bundles")
```

Comment:
Think the bundles tab should be added based on Bundles featureFlag. This code might not break anything as caseBundles would be empty by Bundles tab would be displayed in Tabs.